api/app.go

```diff
@@ -996,11 +996,13 @@ func setEnv(w http.ResponseWriter, r *http.Request, t auth.Token) (err error) {
 		return permission.ErrUnauthorized
 	}
 	var toExclude []string
```

Comment:
@pedrokiefer I want to people use `PruneUnused` in pair of `ManagedBy`, could we have a validation to avoid use of `PruneUnused` without providing a `ManagedBy` ?